app/web-client/fork-ts-checker.config.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2023 SIGHUP s.r.l All rights reserved.
```

Comment:
nitpick: banner should be updated to `2017-present`

Reply:
fixed